chevah/compat
# [Fix #132] Initial py3 migration,

Scope
=====

This is the migration of the compat code to py3.

It removes py2 support as we don't want to look back :)


Changes
=======

NT and Unix services compat were removed as they are no longer used.
We now use external tools to manage our daemons.

NT command line unicode converting code was removed as it's no longer an issue with py3.

The package definition was "modernized" to declare everyting inside setup.cfg

the package was moved outside of the namespace package.
it was also moved into a dedicates 'src' path.

The change was easy as we have already prepared most of the compat code for py3.

How to try and test the changes
===============================

reviewers: @danuker 

check that changes make sense.

We will know for sure if this work, once we integrated it with chevah-server
